test/fixtures/cross_agent_tests/datastores/README.md

```diff
@@ -1,6 +1,6 @@
 ## Datastore instance tests
 
-The datastore instance tests provide attributes similar to what an agent could expect to find regarding a database configuration and specifies the expected [datastore instance metric](https://source.datanerd.us/agents/agent-specs/blob/master/Datastore-Metrics-PORTED.md#datastore-metric-namespace) that should be generated. The table below lists types attributes and whether will will always be included or optionally included in each test case.
+The datastore instance tests provide attributes similar to what an agent could expect to find regarding a database configuration and specifies the expected [datastore instance metric](https://source.datanerd.us/agents/agent-specs/blob/master/Datastore-Metrics-PORTED.md#datastore-metric-namespace) that should be generated. The table below lists types attributes and whether they will always be included or optionally included in each test case.
```

Comment:
atypical correction

Reply:
I'm impressed by its correctness 😄

Reply:
atypical = I intervened carefully instead of just using my default action.

The tooling currently only complains.

Google Sheets will suggest corrections for misspelled words, but while Google Sheets's suggestions are right more often than not but not often enough that I could safely let it run that way (iow, I'd be quickly banned from Hacktoberfest if I did it that way).

I don't have anything that actually suggests any of the other corrections. I'm close to having it actually suggest those other corrections (I have it suggesting patterns for ignoring content...).